sam-template.yaml

```diff
@@ -362,7 +362,9 @@ Resources:
         # TODO create new role without packer permissions
         IamInstanceProfile:
           Name: 'CodeDeploy-EC2-Instance-Profile'
-        ImageId: 'ami-014a706cddda2a79a' # AMI number
+        # output of
+        # https://eu-west-2.console.aws.amazon.com/imagebuilder/home?region=eu-west-2#/images/arn%3Aaws%3Aimagebuilder%3Aeu-west-2%3A732292556707%3Aimage%2Feeimage-ubuntu%2F0.0.35%2F2/details
+        ImageId: 'ami-0c8d319e3a044e7da'
```

Comment:
It seems this project is built off the EE base image, rather than the generic one.
I found this surprising but `ami-014a706cddda2a79a` is definitely an EE image and https://github.com/DemocracyClub/WhoCanIVoteFor/commit/5fa46e477cb9a74ff4d03d429725d78bcda0b13d was the last commit touching this line.

Reply:
So on this one, we want to get to a stage where we are just basing off the generic DC base image, but there are a lot of things in the EE base we need to replicate in order to do it. I think we do want to do this, but I'm going to punt this for another PR.

Reply:
https://app.asana.com/0/1205090401342417/1208112366254722/f

Reply:
When you say "a lot of things"...do you have a list, or just a general sense of dread about it?

Reply:
You mean more than the usual background level of dread? :smile: 

I basically got as far as deciding there were a non-zero number of things, looked at

- https://github.com/DemocracyClub/EveryElection/blob/master/cdk_stacks/components/add_user.yml
- https://github.com/DemocracyClub/EveryElection/blob/master/cdk_stacks/components/install_app.yml
- https://github.com/DemocracyClub/EveryElection/blob/master/cdk_stacks/components/instance_connect.yml
- https://github.com/DemocracyClub/EveryElection/blob/master/cdk_stacks/components/manage_py_command.yml 

and decided it is an issue for another day. Maybe it will be easier than we think when we come to deal with it.